swagger.yaml

```diff
@@ -239,29 +239,28 @@ definitions:
     type: "object"
     required:
       - fiatAccountId
-      - name
-      - institution
+      - accountName
+      - institutionName
       - fiatAccountType
       - fiatAccountSchema
     properties:
       fiatAccountId:
         type: "string"
-      name:
+      accountName:
         type: "string"
-      institution:
+      institutionName:
         type: "string"
       fiatAccountType:
         $ref: '#/definitions/FiatAccountTypeEnum'
       fiatAccountSchema:
         $ref: '#/definitions/FiatAccountSchemaEnum'
   GetFiatAccountsResponse:
     type: object
-    properties:
-      <FiatAccountTypeEnum>:
```

Comment:
this not working is a limitation of the validator package we use?

Reply:
Yeah, that's correct; this is a workaround.